src/lib/utilities/bark-code.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Standard codes for actor actions and server actions.
```

Comment:
Is not just for these right? E.g. we use it in dbResultQuery also. Standard codes for bark bank pawtal. That should do.

Reply:
Fixed 8d4406f